chain/network/src/types.rs

```diff
@@ -236,29 +230,15 @@ impl From<NetworkResponses> for PeerManagerMessageResponse {
 #[allow(clippy::large_enum_variant)]
 pub enum NetworkRequests {
     /// Sends block, either when block was just produced or when requested.
-    Block {
-        block: Block,
-    },
+    Block { block: Block },
     /// Sends approval.
-    Approval {
-        approval_message: ApprovalMessage,
-    },
+    Approval { approval_message: ApprovalMessage },
     /// Request block with given hash from given peer.
-    BlockRequest {
-        hash: CryptoHash,
-        peer_id: PeerId,
-    },
+    BlockRequest { hash: CryptoHash, peer_id: PeerId },
     /// Request given block headers.
-    BlockHeadersRequest {
-        hashes: Vec<CryptoHash>,
-        peer_id: PeerId,
-    },
+    BlockHeadersRequest { hashes: Vec<CryptoHash>, peer_id: PeerId },
     /// Request state header for given shard at given state root.
-    StateRequestHeader {
-        shard_id: ShardId,
-        sync_hash: CryptoHash,
-        target: AccountOrPeerIdOrHash,
-    },
```

Comment:
Note that such formatting changes don't really help with reviewing gigantic PRs :)

As a general wish item, please try to find mode ways to minimize the diffs (not at the expense of landing awesome refactors obviously! ;) )

Reply:
ack